src/Go.CodeAnalysis/Text/SnapshotSegment.cs

```diff
@@ -3,6 +3,9 @@
     using System;
     using System.Text;
 
+    /// <summary>
+    /// Represents a code segment.
```

Comment:
Perhaps ‘code span’? The only reason I didn’t use ‘SnapshotSpan’ is because the Editor already has a type of that name